cogs/birthday/testBirthday.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+import discord
+import pytest
+
+
+from .birthday import Birthday
+from .constants import CANNED_MESSAGES
+
+BOT_BIRTHDAY = "Wow! It's my own birthday! Happy birthday to myself!"
+
+
+class MockUser:
+    def __init__(self, _id: int, mention: str):
+        self.id = _id
+        self.mention = mention
+
+
+class MockBot:
+    user = MockUser(1234, "Ren")
+
+
+# XXX Move this out later once we refactor birthday cog layout
+class TestBirthdayHelpers:
+    bot = MockBot()
+
+    @pytest.mark.parametrize(
+        ["uid", "mention"],
+        [
+            (1111, "Onii-chan"),
+            (2222, "Ara-ara Onee-chan"),
+            (3333, "SENPAI"),
+            (4444, "Sensei"),
+        ],
+    )
+    def testGetBirthdayMessageWhenUserIsNotBot(self, mocker, uid, mention):
+        # Do not initialize config, logger, and background task
+        mocker.patch("birthday.Birthday.initializeConfigAndLogger")
+        mocker.patch("birthday.Birthday.initializeBgTask")
+
+        birthday = Birthday(self.bot)
+        mockUser = MockUser(uid, mention)
+        expectedMessages = [msg.format(mockUser.mention) for msg in CANNED_MESSAGES]
+
+        msg = birthday.getBirthdayMessage(mockUser)
+        assert msg in expectedMessages
+
+    def testGetBirthdayMessageWhenUserIsBot(self, mocker):
+        # Do not initialize config, logger, and background task
+        mocker.patch("birthday.Birthday.initializeConfigAndLogger")
+        mocker.patch("birthday.Birthday.initializeBgTask")
+
+        birthday = Birthday(self.bot)
+        mockUser = MockUser(1234, "Ren")
+
+        msg = birthday.getBirthdayMessage(mockUser)
+        assert msg == BOT_BIRTHDAY
```

Comment:
Perhaps a better name could be used? I mean, it sounds more like this constant is about the bot's birthday than the message sent on the day.
```suggestion
        assert msg == BOT_BIRTHDAY_MSG
```